modules/adding-node-iso-configs.adoc

```diff
@@ -68,6 +68,17 @@ For more information, see "Root device hints" in the "Setting up the environment
 The configuration must match the Host Network Management API defined in the link:https://nmstate.io/[nmstate documentation].
 |A dictionary of host network configuration objects.
 
+|cpuArchitecture
+|Optional.
+Specifies the architecture of the nodes you are adding.
+This parameter allows you to override the default value from the cluster when required.
+|String
+
+|sshKey
+|Optional.
+The file containing the SSH key to authenticate access to your cluster machines.
+|String
```

Comment:
```suggestion
|String.
```

Reply:
I notice in the other table entries, Sting has a `.`, so just add it to these for consistency :)